twine/commands/check.py

```diff
@@ -63,6 +63,13 @@ def __str__(self) -> str:
         return self.getvalue().strip()
 
 
+# from Python 3.11 docs
+def _parse_content_type(value: str) -> Tuple[str, Dict[str, str]]:
```

Comment:
FYI, I tweaked this to be aligned with the other utility methods/classes in this module.